R/processing_pcs.R

```diff
@@ -61,3 +61,68 @@ downsample_pcs <- function(pcs, frac_cells, min_cells = 50) {
 
   return(downsampled_pcs)
 }
+
+#' Calculate within-batch ARI using provided PCs for use in integration metric calculations
```

Comment:
Because this function is only being used inside the other within batch ari function, I would move this to be within the same script, just below the main function.